README.md

```diff
@@ -37,6 +41,17 @@ Install Docker Desktop ([Get Docker](https://docs.docker.com/get-docker/))
 
 `gcloud config set project <project-id>`
 
+## Testing
+
+Unit tests can be run using pytest. Running pytest will recursively look for and execute test files.
+
+```bash
+pip install pytest
+pytest
+```
+
+To test from the packaged version of the ingestion library, run `pip install -e python/` before testing.
```

Comment:
does `-e python/` install all the packages within `python/`?

Reply:
It will install all of the packages defined in setup.py in python/ in edit mode, which means you can edit them without having to reinstall.

Reply:
Wooooah that is cool, I didn't know that